ui/app/src/model/project-client.ts

```diff
@@ -120,7 +120,7 @@ export function useCreateProjectMutation() {
       return createProject(project);
     },
     onSuccess: () => {
-      return queryClient.invalidateQueries([...queryKey]);
+      return Promise.all([queryClient.invalidateQueries([...queryKey]), queryClient.invalidateQueries([userKey])]);
```

Comment:
whye not ``queryClient.invalidateQueries([userKey, ...queryKey])`` ?

Reply:
No valid